make-webpack-config.js

```diff
@@ -25,6 +26,7 @@ module.exports = function(options){
   }
 
   return {
+    devtool: isProd ? 'hidden-source-map' : 'eval',
```

Comment:
Source map are not loaded till you actually open devtools. Thus there is absolutely no impact for regular users. If you open devtools, you probably want to do debugging and then you will feel slight impact for sure.